CHANGELOG.md

```diff
@@ -58,5 +133,51 @@ Changelog
    Damit können Klienten den Baum verändern/erzeugen/ergänzen - und zwar genau in dem Moment, wo er erstmals benötigt wird.
    Details zur Verwendung in 818daa580f243344b183e67438dea3592eb24b8a.
 
+## Version 2
```

Comment:
Viel besser, das hier im Changelog zu haben! (Vermute, es war vorher in der README?)

Reply:
Genau!